libsql/src/local/impls.rs

```diff
@@ -105,6 +105,13 @@ impl Stmt for LibsqlStmt {
             .map(|r| Rows { inner: Box::new(r) })
     }
 
+    async fn run(&mut self, params: &Params) -> Result<()> {
+        let params = params.clone();
```

Comment:
Why do we need to clone the params?

Reply:
@haaawk I don't think it is needed, but as both `execute()` and `query()` do it, I didn't want to break the symmetry... 🤔